Library/Homebrew/tab.rb

```diff
@@ -384,6 +387,7 @@ def to_s
       "Built from source"
     end
 
+    s << "using the API" if loaded_from_api
```

Comment:
```suggestion
    s << "using the formulae.brew.sh API" if loaded_from_api
```

or something?

Reply:
Think it's worth saying `JSON API` so it's not tap-specific in case we expand this to other taps